index.d.ts

```diff
@@ -676,6 +676,27 @@ declare module 'egg' {
 
     truncated: boolean;
   }
+                   
+  interface GetFileStreamOptions {
+    requireFile?: boolean // required file submit, default is true
+    defCharset?: string
+    limits?: {
+      fieldNameSize: number
```

Comment:
fields of `limits` should be optional, btw, plz add semicolon to each property declaration.